src/CopyrighFixer/CopyrighFixer/worker.cpp

```diff
@@ -0,0 +1,42 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the lgplv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#include "config.h"
+#include "worker.h"
+#include <quasarapp.h>
+
+
+
+namespace CopyrighFixer {
+Worker::Worker() {
+
+}
+
+bool Worker::run() {
+    return 1;
```

Comment:
in the current time your application do not support run method. So you need to return false.